docs/qemu.md

```diff
@@ -45,12 +45,5 @@ To mount host folder in QEMU, simply run:
 ```bash
 $ mount -t 9p -o trans=virtio host <mount_point>
 ```
-# 5. SLiRP user networking
-To enable SLiRP user networking just set `QEMU_USERNET_ENABLE ?= y` in `common.mk`.
-After booting QEMU VM, eth0 will automatically receive IP address via DHCP
-
-*Important* Take into account that ICMP doesn't work in SLiRP mode,
```

Comment:
Is this note still relevant?

Reply:
It is. Well, `ping localhost` works, but pinging external hosts does not.